dirWatcher.js

```diff
@@ -0,0 +1,35 @@
+import fs from "fs";
+import EventEmitter from "events";
+
+class DirWatcher extends EventEmitter {
+    constructor() {
+        super();
+    }
+
+    watch(path, delay) {
+        try {
+            let lastModifiedTime;
+            setInterval(() => {
+                let stats = fs.statSync(path);
+
+                if (!stats.isDirectory()) {
+                    throw ("Invalid folder path")
+                }
+                /**To treat new files as not a 'change' */
```

Comment:
The comment is redundant, please keep code clean when you are going to merge it into master branch.

Reply:
Removed the comment. Thanks for the comment.